### PR TITLE
updated history section with query

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

File: web/gatsby-config.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = {
         description: `AIRtec specializes in modifying aircraft to meet unique operational requirements, then operating & maintaining them to deliver outstanding mission results & data.`,
         image:
             'https://cdn.sanity.io/images/wqj2xpad/production/234735b1b8a645eeeaff61265a642fe6c42668f6-4608x2592.jpg',
-        twitter: '',
+        twitter: ' ',
```
Comment (asasmith) [inline web/gatsby-config.js:13]: Is this necessary?

Comment (kcbreuil) [reply web/gatsby-config.js:13]: @asasmith it was throwing an error and the space made it go away?

Comment (asasmith) [reply web/gatsby-config.js:13]: if we're not using a twitter handle we should probably just remove this field

Comment (kcbreuil) [reply web/gatsby-config.js:13]: Ok

**Review by asasmith — APPROVED** [review]

**Pull request merged**